src/localization/de/messages.json

```diff
@@ -336,63 +336,63 @@
+  "interface.premiumPage.unlocks.new": "<0>Neue Sachen</0><1>Nichts entwickelt sich selbst.</1> Deine Einträge werden neue Sachen und Verbesserungen für die Website fundieren, dass sie noch besser wird. Du wirst auch zusätzliche Features freischalten, die bei deiner Analyse helfen.",
   "interface.premiumPage.unlocks.noAds": "<0>Keine Bewerbungen</0><1>Keinem gefallen sie, aber wir brauchen sie.</1> Irgendein Eintrag lohnt sich mehr als die Bewerbungen, also werden wir die Bewerbungen von der Plattform entfernen für dich, dass du unseren Inhalt mit weniger Ablenkungen und Unordnung genießen kannst.",
   "interface.premiumPage.unlocks.updates": "<0>Aktualisierungen für Patches</0><1>Für neue Patches zu aktualisieren ist viel Arbeit.</1> Wir müssen alle Zauber-Änderungen anwenden, neue Eigenschaften hinzufügen, Unterstützung für die neuen Kämpfe hinzufügen, Screenshots machen, Kampfphasen, Buffs und Debuffs hinzufügen und so weiter. Deine Einträge machen es uns möglich, die Spezialisierungen auf dem neuesten Stand zu behalten, als sie in neuen Patches verändert werden.",
   "interface.premiumPage.you": "Du",
-  "interface.report.advancedLoggingWarning.advancedLoggingWarning": "",
-  "interface.report.advancedLoggingWarning.advancedLoggingWarningDetails": "",
-  "interface.report.classicLogWarning.classicEncounter": "",
-  "interface.report.classicLogWarning.classicEncounterDetails": "",
-  "interface.report.fightSelection.documentTitle": "",
-  "interface.report.fightSelection.fightSelection": "",
-  "interface.report.fightSelection.fightSelectionDetails": "",
+  "interface.report.advancedLoggingWarning.advancedLoggingWarning": "Erweitertes Kampflog nicht aktiviert",
+  "interface.report.advancedLoggingWarning.advancedLoggingWarningDetails": "Ohne das erweiterte Kampflog können wir keine kritischen Informationen über Spieler abrufen inklusive Spezialisierungen, Talente und Gegenstände. Ohne diese Informationen können wir diesen Kampf nicht analysieren. Bitte stelle sicher, dass das erweiterte Kampflog aktiviert im Spiel ist (Gefunden in den Spieleinstellungen unter System/Netzerk).",
+  "interface.report.classicLogWarning.classicEncounter": "Classic WoW Kämpfe erkannt",
+  "interface.report.classicLogWarning.classicEncounterDetails": "Der aktuelle Report enthält Kämpfe von World of Warcraft: Classic. WoWAnalyzer unterstützt aktuell keine Classic WoW Logs und hat keine Pläne, Classic WoW Logs zu unterstützen.",
+  "interface.report.fightSelection.documentTitle": "{0} in {1}",
+  "interface.report.fightSelection.fightSelection": "Kampfauswahl",
+  "interface.report.fightSelection.fightSelectionDetails": "Wähle den Kampf aus, den du analysieren möchtest. Wenn ein Boss oder Kampf fehlt, oder die Liste nach unten leer ist, drücke den 'Neu laden'-Knopf nach oben, um das Log erneut von Warcraft Logs abzurufen. Außerdem, bitte notiere, dass wegen der Weise, auf die Kampflogs funktionieren, können wir keine Trainingsattrappe-Logs analysieren.",
```

Comment:
'press the Refresh button above' -> Damit ist gemeint, dass der Knopf über dem Text ist, also eher
`drücke den "Neu laden" oben`